src/Raven.Studio/typescript/commands/database/documents/getDocumentsPreviewCommand.ts

```diff
@@ -28,26 +28,38 @@ class getDocumentsPreviewCommand extends commandBase {
     static readonly ArrayStubsKey = "$a";
     static readonly TrimmedValueKey = "$t";
 
-    constructor(private database: database, private skip: number, private take: number, private collectionName: string, private previewBindings?: string[], private fullBindings?: string[]) {
+    constructor(private database: database, private skip: number, private take: number, private collectionName: string,
+                private previewBindings?: string[], private fullBindings?: string[], private continuationToken?: string) {
         super();
     }
 
     execute(): JQueryPromise<pagedResultWithAvailableColumns<document>> {
-        const args = {
+        
+        const argsWithContinuationToken = {
```

Comment:
@Danielle9897 extract this to separate method. Right now you  create 2 objects and choose one.

Reply:
done